service/worker/deletenamespace/deleteexecutions/workflow.go

```diff
@@ -128,7 +130,7 @@ func DeleteExecutionsWorkflow(ctx workflow.Context, params DeleteExecutionsParam
 		})
 
 		ctx2 := workflow.WithLocalActivityOptions(ctx, localActivityOptions)
-		err := workflow.ExecuteLocalActivity(ctx2, a.GetNextPageTokenActivity, GetNextPageTokenParams{
+		err := workflow.ExecuteLocalActivity(ctx2, localActivityHandles.GetNextPageTokenActivity, GetNextPageTokenParams{
```

Comment:
This still won't use `primitives.DeleteNamespaceActivityTQ` because it is LA, right? I would build `ctx2` from context that doesn't have TQ set.

Reply:
Right, local activities don't use the set activity task queue. Why use a separate context for local activities? I'm not sure I understand the value there.